src/pages/p/[handle].page.tsx

```diff
@@ -1,46 +1,166 @@
+import { useCurrencyConverter } from 'hooks/CurrencyConverter'
 import { useProjectsQuery } from 'hooks/Projects'
+import useSymbolOfERC20 from 'hooks/SymbolOfERC20'
 import useBalanceOfProject from 'hooks/v1/contractReader/BalanceOfProject'
 import useCurrentFundingCycleOfProject from 'hooks/v1/contractReader/CurrentFundingCycleOfProject'
 import useCurrentPayoutModsOfProject from 'hooks/v1/contractReader/CurrentPayoutModsOfProject'
 import useCurrentTicketModsOfProject from 'hooks/v1/contractReader/CurrentTicketModsOfProject'
 import useOverflowOfProject from 'hooks/v1/contractReader/OverflowOfProject'
+import useOwnerOfProject from 'hooks/v1/contractReader/OwnerOfProject'
 import useProjectIdForHandle from 'hooks/v1/contractReader/ProjectIdForHandle'
 import useQueuedFundingCycleOfProject from 'hooks/v1/contractReader/QueuedFundingCycleOfProject'
 import useQueuedPayoutModsOfProject from 'hooks/v1/contractReader/QueuedPayoutModsOfProject'
 import useQueuedTicketModsOfProject from 'hooks/v1/contractReader/QueuedTicketModsOfProject'
-import useSymbolOfERC20 from 'hooks/SymbolOfERC20'
 import useTerminalOfProject from 'hooks/v1/contractReader/TerminalOfProject'
 import useTokenAddressOfProject from 'hooks/v1/contractReader/TokenAddressOfProject'
-import useUriOfProject from 'hooks/v1/contractReader/UriOfProject'
-import { useCurrencyConverter } from 'hooks/CurrencyConverter'
+import { consolidateMetadata, ProjectMetadataV4 } from 'models/project-metadata'
 import { V1CurrencyOption } from 'models/v1/currencyOption'
-import { useMemo } from 'react'
+import { GetStaticPaths, GetStaticProps, InferGetStaticPropsType } from 'next'
 import { useRouter } from 'next/router'
-import { getTerminalName, getTerminalVersion } from 'utils/v1/terminals'
 import { V1CurrencyProvider } from 'providers/v1/V1CurrencyProvider'
+import { useMemo } from 'react'
+import { ipfsCidUrl } from 'utils/ipfs'
 import { V1CurrencyName } from 'utils/v1/currency'
-import NewDeployNotAvailable from 'components/NewDeployNotAvailable'
-import Project404 from 'components/Project404'
-import { usePageTitle } from 'hooks/PageTitle'
+import ScrollToTopButton from 'components/ScrollToTopButton'
 
-import useOwnerOfProject from 'hooks/v1/contractReader/OwnerOfProject'
+import { getTerminalName, getTerminalVersion } from 'utils/v1/terminals'
+import { paginateDepleteProjectsQueryCall } from 'utils/apollo'
+
+import Loading from '../../components/Loading'
+import V1Project from '../../components/v1/V1Project'
 
-import { layouts } from 'constants/styles/layouts'
 import { projectTypes } from 'constants/v1/projectTypes'
+import { layouts } from 'constants/styles/layouts'
 import { V1ArchivedProjectIds } from 'constants/v1/archivedProjects'
 
-import Loading from '../../components/Loading'
-import V1Project from '../../components/v1/V1Project'
+export const getStaticPaths: GetStaticPaths = async () => {
+  if (process.env.BUILD_CACHE_V1_PROJECTS === 'true') {
```

Comment:
**question** this is cool. When do we enable it? only on mainnet (jb.money)?

Reply:
I currently have V1 disabled everywhere, but easy enough to turn on. It's ultimately up to you guys, but I wonder how much optimisation we should do around v1

Reply:
Whats the downside of enabling it always?

Actually, feasible to hardcode `staticPaths`. We can probably get away with hardcoding:
- jbdao v1
- assangedao
- constitution
- peel
- wagmi
- warren buffet lunch
- lexicon
- ...

Basically just the ones with any sense of a pulse.

Reply:
yeah we can definitely hardcode, although tbh I think we should move away from hard code. Even an env var is likely better than a hardcode. DB is obviously the best. just so we have a way to manipulate outside of a PR